editor/components/inserter/index.js

```diff
@@ -51,8 +56,9 @@ class Inserter extends wp.element.Component {
 					onClick={ this.toggle }
 					className="editor-inserter__toggle"
 					aria-haspopup="true"
+					buttonRef={ ( node ) => this.nodes.toggle = node }
```

Comment:
Yeah that is fine too, in the future though if other refs need to be used for inserter it is nice to give them a little namespace in nodes.  Can change though let me know what you think.